extensions/sns/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 <!-- next-header -->
 
 ## [Unreleased] - ReleaseDate
+- The behavior of the `dfx sns` extension has been updated to match the `sns-cli` tool.
+  The main effect of this is that many deprecated commands have been removed, and the remaining commands have been renamed.
+- Renamed `dfx sns config` to `dfx sns init-config-file`
+- Removed `dfx sns config create`
+- Removed `dfx sns deploy`
```

Comment:
Reading this, I wonder what I should do if I have something that calls `dfx sns config create` or `dfx sns deploy`.  I suppose `deploy-testflight` would be where to look for a replacement for `dfx sns deploy`.  What about `dfx sns config create` ?

Reply:
The current recommendation is to look at [this](https://github.com/dfinity/sns-testing/blob/main/example_sns_init.yaml) as a reference. I just updated the changelog to point towards it. 

`deploy-testflight` serves a different role than `deploy` used to. There's not really anything directly analagous to `deploy` now.